src/sensors/bno055sensor.cpp

```diff
@@ -28,7 +28,7 @@
 void BNO055Sensor::motionSetup() {
     imu = Adafruit_BNO055(sensorId, addr);
     delay(3000);
-    if (!imu.begin(Adafruit_BNO055::OPERATION_MODE_IMUPLUS))
+    if (!imu.begin(Adafruit_BNO055::OPERATION_MODE_NDOF))
```

Comment:
Using the magnetometer isn't always desirable, this could cause issues with people who have high magnetic interference. This setting could likely be tied to one of the magnetometer options in the debug.h file to allow for enabling/disabling it by user choice.

Reply:
True! Forgot about that.